pkg/models/datasource_cache.go

```diff
@@ -75,7 +75,8 @@ func (ds *DataSource) HTTPClientOptions() sdkhttpclient.Options {
 	tlsOptions := ds.TLSOptions()
 	opts := sdkhttpclient.Options{
 		Timeouts: &sdkhttpclient.TimeoutOptions{
-			Timeout:               ds.getTimeout(),
+			Timeout:               time.Duration(setting.DataProxyTimeout) * time.Second,
```

Comment:
This should still use `ds.getTimeout` I believe and if not set in data source config it will fallback to `setting.DataProxyTimeout`.